cocos/asset/assets/simple-texture.ts

```diff
@@ -322,7 +322,9 @@ export class SimpleTexture extends TextureBase {
         if (this._width === 0 || this._height === 0) { return; }
         let flags = TextureFlagBit.NONE;
         if (this._mipFilter !== Filter.NONE && canGenerateMipmap(device, this._width, this._height)) {
-            this._mipmapLevel = getMipLevel(this._width, this._height);
+            if (!this.isCompressed) {
+                this._mipmapLevel = getMipLevel(this._width, this._height);
+            }
```

Comment:
Is this `compressed` behavior appropriate? @yiwenxue

Reply:
fixed